Sensors.cpp

```diff
@@ -30,7 +25,7 @@ void Sensors::loop() {
     if ((millis() - pmLoopTimeStamp > sample_time * 1000)) {  // sample time for each capture
         dataReady = false;
         pmLoopTimeStamp = millis();
-        am2320Read();
+        H&TRead();
```

Comment:
Please rename this method (H&TRead) according [Google Style Guide](https://google.github.io/styleguide/cppguide.html#Function_Names)